nixos/modules/services/mail/mailman.nix

```diff
@@ -383,6 +395,11 @@ in {
       with open('/var/lib/mailman-web/settings_local.json') as f:
           globals().update(json.load(f))
 
+      ${optionalString (cfg.restApiPassFile != null) ''
+        with open('${cfg.restApiPassFile}') as f:
+            MAILMAN_REST_API_PASS = f.read().rstrip('\n')
```

Comment:
could this bit use `LoadCredential`? would also not need the config munging below

Reply:
This will have the downside that we'd have to implement LoadCredential= for each service that is involved here, so I did this for simplicity and also consistency w/bindPassword for LDAP.

if we want to integrate LoadCredential here, this should be done in its own PR IMHO.